python/lsst/display/matplotlib/matplotlib.py

```diff
@@ -600,6 +734,9 @@ def _pan(self, colc, rowc):
     def _getEvent(self, timeout=-1):
         """Listen for a key press, returning (key, x, y)"""
 
+        if timeout < 0:
+            timeout = 24*3600           # -1 generates complaints in QTimer::singleShot
```

Comment:
Do we want to comment why 1 day is a good timeout number?

Reply:
It just needs to be a long time;  I added that